Dockerfile

```diff
@@ -1,14 +1,10 @@
-# Do not upgrade to alpine 3.13 as its nslookup tool returns 1, instead of 0
```

Comment:
alpine 1.14 suffers from the same issue.

Reply:
How does it affect Hubble CLI?

Reply:
It probably doesn't, but the commit description should be updated :-)

Reply:
As this is a backport PR, I don't want to amend the commit. I'll pull in #587 once it's merged into `master`.

Reply:
I missed it was a backport PR because there are no references to upstream commits :scream_cat:

Reply:
Well, the process is a little different for Hubble CLI. There's v0.8 in the PR title, a `backport/0.8` label and a mention of the backported PR in the PR message. Isn't this sufficient?

Reply:
No, no, that's fine :smile_cat: Having that information in the git history is just convenient to avoid going through GitHub all the time.